recipes/wordreference

```diff
@@ -0,0 +1 @@
+(wordreference :fetcher git :url  "https://codeberg.org/martianh/wordreference.el" :files ("wordreference.el"))
```

Comment:
This is equivalent and gives the package room to grow:
```suggestion
(wordreference :url "https://codeberg.org/martianh/wordreference.el" :fetcher git)
```

Reply:
thanks for taking a look at the package. 

> I'm curious about the name of the package although I haven't tried it -- this says it's an interface to wordreference.com but on a quick glance I'm also seeing functions for linguee.com and cntrl.fr. Should the summary of the package be expanded to mention these?

the functions for those other dictionaries just launch a search in them for the same term searched in wordreference. there's also a half-broken one for `helm-dictionary`.

wordreference.com is first point of call, but if you don't find decent info you go scrounging around in various other dictionaries. it's handy to have 3-4 available, and fast.

i use the same workflow in `emacs-leo`, which has bindings for optional second search in say `dictcc.el`.